src/app/mandate/mandate.component.ts

```diff
@@ -0,0 +1,51 @@
+import { Component } from '@angular/core';
+import {ComponentsModule} from "@biggive/components-angular";
+import {DatePipe} from "@angular/common";
+import {ExactCurrencyPipe} from "../exact-currency.pipe";
+import {FaIconComponent} from "@fortawesome/angular-fontawesome";
+import {MatProgressSpinner} from "@angular/material/progress-spinner";
+import {Mandate} from "../mandate.model";
+
+@Component({
+  selector: 'app-mandate',
+  standalone: true,
+  imports: [
+    ComponentsModule,
+    DatePipe,
+    ExactCurrencyPipe,
+    FaIconComponent,
+    MatProgressSpinner
+  ],
+  templateUrl: './mandate.component.html',
+  styleUrl: './mandate.component.scss'
+})
+export class MandateComponent {
+  personId = 'donor-id';
+  complete = true;
+  encodedShareUrl = '';
+  encodedPrefilledText: string = '';
+  donation: any = {};
+  totalValue: number = 20;
+  giftAidAmount: number = 0;
+  totalPaid: 0;
+  mandate: Mandate = {
+  id: 'f7037101-b555-4482-afff-43145fac78bb',
+  campaignId: 'a056900002TPVz5AAH',
+  charityName: '0011r00002Hoe8lAAB',
+  matchingPeriodText: 'for the next three months',
```

Comment:
This text is written as half a sentence, which means we'd end up with half of a piece of microcopy in matchbot and half in frontend ("Your donation of
              {amount} is worth {doubleAmount} for the next three months").
              
I don't think we should do that, it's going to make it complicated to edit that copy having half in one place and half in another. Even more so since the two apps get deployed separately.

Not sure what the best solution is, but maybe this?

```suggestion
  numberOfDonationsMatched: 3,
```

Also since this page can be viewed any time it won't always be the next three months, so maybe we should change the copy to say the first three donations will be matched instead of donations will be matched for the next three months?

Reply:
changed in last commit!